ci-operator/step-registry/upi/conf/gcp/xpn/upi-conf-gcp-xpn-commands.sh

```diff
@@ -24,15 +24,22 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-HOST_PROJECT="openshift-dev-installer"
-CLUSTER_NETWORK="https://www.googleapis.com/compute/v1/projects/openshift-dev-installer/global/networks/installer-shared-vpc"
-CONTROL_SUBNET="https://www.googleapis.com/compute/v1/projects/openshift-dev-installer/regions/us-east1/subnetworks/installer-shared-vpc-subnet-1"
-COMPUTE_SUBNET="https://www.googleapis.com/compute/v1/projects/openshift-dev-installer/regions/us-east1/subnetworks/installer-shared-vpc-subnet-2"
-COMPUTE_SERVICE_ACCOUNT="do-not-delete-ci-xpn@openshift-gce-devel-ci.iam.gserviceaccount.com"
-CONTROL_SERVICE_ACCOUNT="do-not-delete-ci-xpn@openshift-gce-devel-ci.iam.gserviceaccount.com"
-PRIVATE_ZONE_NAME="ci-op-xpn-private-zone"
+if [[ -f "${CLUSTER_PROFILE_DIR}/xpn_project_setting.json" ]]; then
 
-cat > "${SHARED_DIR}/xpn.json" << EOF
+    echo "Using QE XPN configurations..."
```

Comment:
How about `echo "Using XPN project configurations from ${CLUSTER_PROFILE} ..."`

Reply:
Ever tried with "${CLUSTER_PROFILE}", but it's undefined.

Reply:
Then we can remove the var in the echo command